datafusion-cli/src/print_options.rs

```diff
@@ -73,6 +77,8 @@ pub struct PrintOptions {
     pub quiet: bool,
     pub maxrows: MaxRows,
     pub color: bool,
+    pub object_store_profile_mode: InstrumentedObjectStoreMode,
+    pub instrumented_registry: Arc<InstrumentedObjectStoreRegistry>,
```

Comment:
I don't love carrying this as a public member of this struct, but I didn't want to go making too many unnecessary changes without some additional reviewer input. I feel like `PrintOptions` might be to the point where it needs a builder or perhaps a dedicated `new()` method to better encapsulate its behavior.

Reply:
Yes, I agree it is time to encapsulate it